### PR TITLE
Stop passing blob_index unnecessarily into ReplayStage

### PR DESCRIPTION
**Pull request by mvines**
Figure it out yourself son.

### CODE REVIEW

File: src/tpu.rs [hunk 1]
```diff
@@ -225,6 +224,11 @@ impl Tpu {
         // TODO: Fix BankingStage/BroadcastService to operate on `slot` directly instead of
         // `max_tick_height`
         let max_tick_height = (slot + 1) * leader_scheduler.read().unwrap().ticks_per_slot - 1;
+        let blob_index = if let Some(meta) = blocktree.meta(slot).expect("Database error") {
```
Comment (garious) [inline src/tpu.rs]: While you're in the neighborhood?

```rust
let blob_index = blocktree.meta(slot).expect("Database error").map(|meta| meta.consumed).or(0);
```

Comment (mvines) [reply src/tpu.rs]: 10-4

File: src/tpu.rs [hunk 2]
```diff
@@ -225,6 +224,12 @@ impl Tpu {
         // TODO: Fix BankingStage/BroadcastService to operate on `slot` directly instead of
         // `max_tick_height`
         let max_tick_height = (slot + 1) * leader_scheduler.read().unwrap().ticks_per_slot - 1;
+        let blob_index = blocktree
+            .meta(slot)
+            .expect("Database error")
+            .map(|meta| meta.consumed)
+            .or(Some(0))
```
Comment (garious) [inline src/tpu.rs]: oops, `unwrap_or(0)`

Comment (mvines) [reply src/tpu.rs]: Ok now I think you're just trying to get ahead of me in CI

Comment (garious) [reply src/tpu.rs]: ha!

**Pull request merged**